src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/repository/BedSearchRepository.kt

```diff
@@ -365,7 +365,7 @@ class TemporaryAccommodationBedSearchResult(
   bedId: UUID,
   bedName: String,
   roomCharacteristics: MutableList<CharacteristicNames>,
-  val overlaps: MutableList<TemporaryAccommodationBedSearchOverlapResult>,
+  val overlaps: MutableList<TemporaryAccommodationBedSearchResultOverlap>,
```

Comment:
Another git suggestion: given we haven't merged this in and it's part of our original proposal I'd have considered using [git absorb](https://github.com/tummychow/git-absorb) to rework this renaming back into your previous commits. It makes the change set more concise, easy to read & review and easier to revert.

Reply:
I'd really like to get this absorbed into previous commits, but I'm not 100% sure if it's possible to run git absorb across things which have already been committed. I *could* cherry-pick each change into its individual past commit, but I'd be concerned about how much time that might take, given that this is still blocking a front-end ticket.

Reply:
I had a bit of a brainwave this morning and have now successfully absorbed these later clarifying commits into the previous ones.